cache/remotecache/s3/s3.go

```diff
@@ -446,7 +446,7 @@ func (s3Client *s3Client) touch(ctx context.Context, key string) error {
 		Bucket:            &s3Client.bucket,
 		CopySource:        &copySource,
 		Key:               &key,
-		Metadata:          map[string]string{"updated_at": time.Now().String()},
+		Metadata:          map[string]string{"updated-at": time.Now().String()},
```

Comment:
I think we should just remove this metadata. Don't think it has any purpose for BuildKit. cc @bpaquet.

Reply:
No, to do a touch, we need to change a meta data. No issue to change the name :)